sov-modules/sov-modules-impl/bank/src/call.rs

```diff
@@ -1,56 +1,102 @@
-use anyhow::Result;
+use crate::{Amount, Bank, Coins, Token};
+use anyhow::{bail, Result};
 use sov_modules_api::CallResponse;
 use sov_state::WorkingSet;
 
-use crate::{Amount, Bank, Coins};
-
+/// This enumeration represents the available call messages for interacting with the bank module.
 #[derive(borsh::BorshDeserialize, borsh::BorshSerialize, Debug, PartialEq)]
 pub enum CallMessage<C: sov_modules_api::Context> {
+    /// Creates a new token with the specified name and initial balance.
     CreateToken {
+        /// salt: a random value use to create a unique token address.
```

Comment:
(question, non-blocking): Is there an advantage to starting each comment with the name of the field?

Reply:
Not really I can remove it